src/overlays/actors/ovl_Dm_Opstage/z_dm_opstage.c

```diff
@@ -16,7 +16,14 @@ void DmOpstage_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void DmOpstage_Update(Actor* thisx, GlobalContext* globalCtx);
 void DmOpstage_Draw(Actor* thisx, GlobalContext* globalCtx);
 
-void func_80A9FA58(DmOpstage* this, GlobalContext* globalCtx);
+void DmOpstage_FollowCutsceneScript(DmOpstage* this, GlobalContext* globalCtx);
+
+typedef enum {
+    /* 0 */ DM_OPSTAGE_TYPE_FLOOR,
+    /* 1 */ DM_OPSTAGE_TYPE_TREE1,
+    /* 2 */ DM_OPSTAGE_TYPE_TREE2,
+    /* 3 */ DM_OPSTAGE_TYPE_TREE3,
+} DmOpStageTypes;
```

Comment:
This is something that you'd want to access when spawning the actor, so in the header please.